Security/Web_Security.mediawiki

```diff
@@ -0,0 +1,614 @@
+<div style="max-width: 75em;">
+<table>
+  <tr>
+    <td style="white-space: nowrap;">__TOC__</td>
+    <td style="vertical-align: top; padding-left: 1em;">The goal of this document is to help operational teams with creating secure web applications. All Mozilla sites and deployments are expected to follow the recommendations below.
+
+The Enterprise Information Security (EIS) team maintains this document as a reference guide to navigate the rapidly changing landscape of web security. Changes are reviewed and merged by the OpSec team, and broadcast to the various Operational teams.
+
+Updates to this page should be submitted to the [https://github.com/mozilla/wikimo_opsec source repository on github].
+    </td>
+  </tr>
+</table>
+
+= Transport Layer Security =
+
+== HTTPS ==
+
+Transport Layer Security (TLS/SSL) provides assurances about the confidentiality, authentication, and integrity of all communications both inside and outside of Mozilla. To protect our users and networked systems, all systems must use encrypted TLS connections for their network communications.  Support and use of TLS is mandatory for all new systems, and existing systems are expected to add support for exclusive TLS as soon as possible.
+
+Websites or API endpoints that only communicate with modern browsers and systems should use the [[Security/Server Side TLS#Modern compatibility|Mozilla modern TLS configuration]].
+
+Websites intended for general public consumption should use the [[Security/Server Side TLS#Intermediate compatibility (default)|Mozilla intermediate TLS configuration]].
+
+Websites that require backwards compatibility with extremely old browsers may use the [[Security/Server Side TLS#Old backward compatibility|Mozilla backwards compatible TLS configuration]]. Very few sites should require this compatibility level, and use of this compatibility level should be noted in your risk assessment.
+
+=== Compatibility ===
+
+{| class="wikitable" style="width: 100%;"
+|-
+! Configuration
+! Oldest compatible clients
+|- style="background-color: #9EDB58;"
+| align="center" | Modern
+| style="padding-left: .5em;" | Firefox 27, Chrome 22, Internet Explorer 11, Opera 14, Safari 7, Android 4.4, Java 8 
+|- style="background-color: #E8E27A;"
+| align="center" | Intermediate
+| style="padding-left: .5em;" | Firefox 1, Chrome 1, Internet Explorer 7, Opera 5, Safari 1, Internet Explorer 8 (XP), Android 2.3, Java 7 
+|- style="background-color: #CCCCCC;"
+| align="center" | Backwards
+| style="padding-left: .5em;" | Internet Explorer 6 (XP), Java 6 
+|}
+
+=== See Also ===
```

Comment:
l43 nitpick:
since these are directly linked, the see also could be at the bottom of the page for readability purposes akin to https://wiki.mozilla.org/Security/Guidelines/OpenSSH#Appendixes